app/components/form-elements.tsx

```diff
@@ -0,0 +1,39 @@
+import * as React from 'react'
+
+function Label(props: JSX.IntrinsicElements['label']) {
+  return (
+    // eslint-disable-next-line jsx-a11y/label-has-associated-control
```

Comment:
Maybe we should disable this rule. It's a pain and rarely actually helps in real-world situations like this.

Reply:
Done